01-main/packages/tixati

```diff
@@ -1,7 +1,7 @@
 DEFVER=1
-get_website "https://www.tixati.com/download/linux.html"
+get_website "https://www.tixati.com/linux"
 if [ "${ACTION}" != "prettylist" ]; then
-    URL="$(grep "amd64\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f2)"
```

Comment:
@philclifford why move away from `"${CACHE_FILE}"`?

Reply:
Unintended consequence of tired and hurried attempt to rapidly fix what I broke while also fixing breakages from upstream.   
Sorry.